honeybee_radiance_command/pinterp.py

```diff
@@ -0,0 +1,124 @@
+"""pinterp command."""
+
+from .options.pinterp import PinterpOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+
+
+class Pinterp(Command):
+    """Pinterp command.
+
+    Pinterp interpolates or extrapolates a new view from one or more RADIANCE pictures 
+    and sends the result to the standard output.
+
+    Args:
+        options: Command options. It will be set to Radiance default values
+            if unspecified.
+        output: File path to the output file (Default: None).
+        view: View to interpolate or extrapolate (Default: None).
+        image: Radiance pictures to interpolate or extrapolate from (Default: None).
+        zspec: The distance to each pixel in the image(s) (Default: None).
```

Comment:
This `zspec` description could be a lot better. Distance from what? Is it always an integer? It looks like it can also be a list of integers?

Reply:
I agree that it is not very helpful now that you point it out. I made a more descriptive text which should hopefully make it more clear.